.github/workflows/scripts/install.sh

```diff
@@ -98,7 +98,7 @@ if [ "$TEST" = "s3" ]; then
   sed -i -e '$a s3_test: true\
 minio_access_key: "'$MINIO_ACCESS_KEY'"\
 minio_secret_key: "'$MINIO_SECRET_KEY'"\
-pulp_scenario_settings: null\
+pulp_scenario_settings: {"domain_enabled": true}\
```

Comment:
You should not manually edit this file. We want to update the plugin template configuration (pulp_settings_s3) and then re-generate the workflow scripts (by running `/plugin-template --github pulp_ostree`).

https://github.com/pulp/plugin_template/?tab=readme-ov-file#add-ci-configuration-to-a-pulp-plugin

https://github.com/pulp/pulpcore/blob/a33612b89f48985448f5315dc5c67aa54f94d196/template_config.yml#L70

Reply:
You still did not update the template configuration and did not run the `plugin-template` utility to re-generate the workflow scripts.